packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -266,7 +271,7 @@ class AccountSwitcher extends React.Component {
 
         const demo_accounts = (
             <div className='acc-switcher__list-wrapper'>
-                <>
+                <React.Fragment>
```

Comment:
do we need this line? it only has one child

Reply:
Yeah, good spot. Missed this as I simply changed all the `<>` to `Fragments` in this file because of the red highlights when viewed in Github.

Reply:
Nope, I just renamed the `<>` into `React.Fragments` so I didn't realize this particular block only has one child.